analytics.js

```diff
@@ -853,7 +853,14 @@ class Analytics {
             }
           }
         } catch (err) {
-          err.message = `[sendToNative]:: [Destination: ${obj.name}]:: "${err.message}"`;
+          if (err instanceof Error) {
+            err.message = `[sendToNative]:: [Destination: ${obj.name}]:: "${err.message}"`;
```

Comment:
Can we avoid repetition of this message format?

Reply:
Addressed.